### DISCUSSION TITLE
集群单节点故障，是否会自动转移到其他节点

### DISCUSSION THREAD

**Discussion by sunnyinchina** [emqx/emqx#8608] [Q&A]
1、问题一
情况如下，A客户端连接到emqx 集群（包括B和C）中B节点，未使用负载均衡器，C节点订阅了B节点的aaa主题，
若B节点宕机了，那么A客户端是否还可发布消息aaa到集群，C节点是否还可以收到主题aaa的消息？

2、问题二
如何通过IP地址列表访问集群？

![image](https://user-images.githubusercontent.com/46027205/181731100-238ae1ba-e9d3-4855-8433-ab14ed30af12.png)

**Comment by lafirest** [comment]
1. If node B is crashed, Client A and node C will lost the connection of node B
2. the IP address is managed by your client, you can pick one of them to connect
